package.json

```diff
@@ -50,7 +50,7 @@
     }
   },
   "lint-staged": {
-    "*.{js,css,json,md}": [
+    "src/**/*.{js,css,json,md}": [
```

Comment:
It'd be a good idea to split changes to `.prettierignore` and `package.json` into a new PR. This one is pretty big.